pkg/linear-pools/contracts/midas-linear-pool/MidasLinearPool.sol

```diff
@@ -73,13 +67,13 @@ contract MidasLinearPool is LinearPool, Version {
     }
 
     function _getWrappedTokenRate() internal view override returns (uint256) {
-        try _cToken.exchangeRateStored() returns (uint256 rate) {
-            // exchangeRateStored returns the exchange rate scaled by 1e18, so no additional
-            // operations are needed here.
+        // Midas' `exchangeRateHypothetical` returns the exchangeRate for the current block scaled to 18 decimals. It
+        // builds on Compounds' `exchangeRateStored` function by projecting the exchangeRate to the current block.
+        try ICToken(address(getWrappedToken())).exchangeRateHypothetical() returns (uint256 rate) {
```

Comment:
`exchangeRateHypothetical` is deprecated, you should use `exchangeRateCurrent` instead...  but if too hard, we can add `exchangeRateHypothetical` back, it is not that big of an issue